content/commitees/specification/operations.md

```diff
@@ -0,0 +1,482 @@
+rules described in the
+<https://wiki.eclipse.org/JakartaEE_Maven_Versioning_Rules> document.
+
+### Java Package # {#java_package}
+
+All new classes, as well as modifications to `javax.` classes, are
+created in the `jakarta.{abbreviation}.</emphasis>` package.
+
+This also applies to OSGi bundles produced by the project.
+
+### Exceptions # {#exceptions}
+
+Petition the specification committee to request an exception to the
+namespace rules above.
+
+## Coding Conventions # {#coding_conventions}
+
+Where feasible, all source content must include a valid copyright and
+license header. Tools such as the `glassfish-copyright-maven-plugin` may
+be useful in ensuring consistency.
+
+## Specification Document Conventions (Currently under discussion) # {#specification_document_conventions_currently_under_discussion}
+
+Written in the third person tense
+
+TBD \"uses\" vs. \"includes\"
+
+Recommended formats, in order of preference: asciidoc, markdown, text.
+
+The first reference to other specifications must use the official full
+name. Subsequent references may use a generally accepted abbreviation.
+
+## Initial Migration Tasks # {#initial_migration_tasks}
+
+The requirements for the initial migration from the legacy Java EE
+projects and associated naming conventions to the Jakarta conventions is
+outlined in these documents:
+
+-   [Oracle/JCP Acronym
+    Guidelines](https://jakarta.ee/legal/acronym_guidelines/)
+
+-   [Project Names and Codes](names.xml)
+
+Using these conventions, projects need to:
+
+-   Change the specification name to that given in [Project Names and
+    Codes](names.xml)
+
+-   JavaDocs and text documents such as README files need to be updated
+    to use these naming conventions.
+
+-   Replace references to JCP process with references to [Eclipse
+    JESP](https://jakarta.ee/about/jesp/)
+
+-   Update references to other specifications to use name in [Project
+    Names and Codes](names.xml)
+
+-   Links to JCP JSR pages need to be replaced with a links of the form
+    jakarta.ee/specifications/\<code\>/\<version\> where \<code\> is the
+    specification code from [Project Names and Codes](names.xml), and
+    \<version\> is the particular specification version.
+
+-   Leave existing \"\@since XYZ 1.x\" uses alone. These refer to the
+    old JCP version. Future additions should use the Jakarta project
+    name.
+
+Specification projects need issue templates and labels to support the
+TCK challenge process, and the compatibility request process. Creating
+an issue template is described in this GitHub doc:
+<https://help.github.com/en/articles/creating-issue-templates-for-your-repository>.
+Creating labels is described in this GitHub doc:
+<https://help.github.com/en/articles/creating-a-label>.
+
+The current TCK challenge and compatibility request process requires the
+following issue labels, which should be defined in the issue tracker:
+
++-----------------------------------+-----------------------------------+
```

Comment:
This table doesn't render correctly on the current version of Hugo (https://deploy-preview-590--jakartaee.netlify.com/commitees/specification/operations/). We've done an upgrade that has slightly shifted how Markdown is processed to be more in line with how [GitHub processes markdown](https://github.github.com/gfm/#tables-extension-), so you'll need to make a couple of tweaks to the table text to get it to render properly. If you like, I can swing by and help you make the changes you need!

Reply:
Thanks! I saw that and fixing it now...